sources/packages/backend/apps/db-migrations/src/sql/InstitutionLocations/Add-col-has-integration.sql

```diff
@@ -0,0 +1,7 @@
+-- Add has_integration to locations table
+ALTER TABLE
+  sims.institution_locations
+ADD
+  COLUMN IF NOT EXISTS has_integration BOOLEAN NOT NULL DEFAULT FALSE;
+
+COMMENT ON COLUMN sims.institution_locations.has_integration IS 'has_integration identifies if the institution uses integration';
```

Comment:
Please adjust the comment to something like `Identifies if the location uses integrations.`

Reply:
Adding the period, please.